components/todo-item.js

```diff
@@ -1,11 +1,50 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem(props) {
+	const { id, date, todo, onRemove } = props;
+
+	return (
+		<TodoItemWrapper>
+			<Todo>{todo}</Todo>
+			<Date>
+				{date}
+				<Button onClick={() => onRemove(id)}>완료</Button>
+			</Date>
+		</TodoItemWrapper>
+	);
 }
```

Comment:
```suggestion
	const { todo, onDelete } = props;
        const { date, todo } = todo;
	return (
		<Wrapper>
		    <Row>
			<P>{todo}</P>
			<P>{date}</P>
		    </Row>
                    <DeleteButton onClick={onDelete}>완료</Button>
		</Wrapper>
	);
}
```
1. ```props, todos```에서 왜 위와 같이 뽑는지는 ```components/todo-list```의 리뷰 확인해주세용
2. 컴포넌트 네이밍 바뀐 거는 아래 styled-component 리뷰들 참고해주세용
3. 날짜를 나타내는 태그 안에 버튼이 있는 게 어색해요. 위와 같은 코드가 어떻게 작동하는지는 아래 stylec-component 리뷰 참고해주세용.
4. 맨 바깥 태그는 Wrapper라고 네이밍 해주시면 됩니다.